myhoard/controller.py

```diff
@@ -308,10 +308,6 @@ def collect_binlogs_to_purge(
                 can_purge = all(
                     stream.is_binlog_safe_to_delete(binlog, exclude_uuid=exclude_uuid) for stream in backup_streams
                 )
-                if can_purge:
-                    log.info("Binlog %s is reported safe to delete by all backup streams", binlog["local_index"])
-                else:
-                    log.info("Binlog %s reported not safe to delete by some backup streams", binlog["local_index"])
```

Comment:
I'd leave the "not safe to delete" logging, that can be informative and shouldn't occur so often.